src/Infrastructure.Dapper/Repositories/UserRepository.cs

```diff
@@ -167,7 +167,19 @@ public override async Task DeleteAsync(User user)
         {
             await connection.ExecuteAsync(
                 $"[{Schema}].[{Table}_DeleteById]",
-                new { Id = user.Id },
+                new { Ids = new List<Guid> { user.Id }.ToGuidIdArrayTVP() },
```

Comment:
Revert this change to pass the single Id. The failing integration test will pass then.